index.html

```diff
@@ -51,21 +49,21 @@ <h2>Social Media Marketing</h2>
     <div class="benefits">
         <div class="benefit-lead">
```

Comment:
The `css` class names can be refactored in to a single class as the css properties for each of the `.benefit` class is the same. Try to reduce code duplication in the style sheet and see how you can generalise the class names here for the benefits